test/controllers/user_controller_test.exs

```diff
@@ -33,7 +33,8 @@ defmodule PhoenixTemplate.UserControllerTest do
   end
 
   test "shows chosen resource", %{conn: conn} do
-    user = Repo.insert! %User{}
+    changeset = User.changeset(%User{}, @valid_create_attrs)
```

Comment:
I can refactor it into its own function but you'll still be calling the function every time. The line count and DRYness won't change as you can't just mutate state like that unless you use macros or some magical thing